feconf.py

```diff
@@ -1293,3 +1293,7 @@ def get_empty_ratings():
 # Minimum timestamp difference required between UserSettingsModel's created_on
 # and some other attribute to use it for updating the created_on attribute.
 CREATED_ON_TIME_DELTA_MINUTES = 5
+
+# Date after which UserSettingModel's created_on attribute is supposed to have
+# its correct value.
+CREATED_ON_CORRECTION_CUTOFF_DATE = 'Jul 1 2020'
```

Comment:
I would place this in the job itself since the job is just temporary and we do not want this constant to be used anywhere else.

Reply:
Okay then, since I am using it only in the job and in one other test `test_update_for_user_model_created_before_cutoff_date`, I have used the string itself directly.